CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+- Guarantee that the `refetchVariables` will follow the `search-protocol`.
+- Send an empty filter when the `facets` query is not ready.
+- Funcion `buildSelectedFacetsAndFullText` now cases where map is lesser than query.
```

Comment:
typo, and this phrase can be better worded, I am having a hard time to understand it